samples/drivers/led_strip/README.rst

```diff
@@ -46,37 +62,13 @@ property. See the overlay file
 Building and Running
 ********************
 
-.. _blog post on WS2812 timing: https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/
-
-This sample's source directory is :zephyr_file:`samples/drivers/led_strip/`.
-
-To make sure the sample is set up properly for building, you must:
-
-- select the correct WS2812 driver backend for your SoC. This currently should
-  be :kconfig:option:`CONFIG_WS2812_STRIP_SPI` unless you are using an nRF51 SoC, in
-  which case it will be :kconfig:option:`CONFIG_WS2812_STRIP_GPIO`.
-  For the nRF52832, the SPI peripheral might output some garbage at the end of
-  transmissions, and that might confuse older WS2812 strips. Use the I2S driver
-  in those cases.
-
-- create a ``led-strip`` :ref:`devicetree alias <dt-alias-chosen>`, which refers
-  to a node in your :ref:`devicetree <dt-guide>` with a
-  ``worldsemi,ws2812-i2s``, ``worldsemi,ws2812-spi`` or
-  ``worldsemi,ws2812-gpio`` compatible. The node must be properly configured for
-  the driver backend (I2S, SPI or GPIO) and daisy chain length (number of WS2812
-  chips).
-
-For example devicetree configurations for each compatible, see
-:zephyr_file:`samples/drivers/led_ws2812/boards/thingy52_nrf52832.overlay`,
-:zephyr_file:`samples/drivers/led_ws2812/boards/nrf52dk_nrf52832.overlay` and
-:zephyr_file:`samples/drivers/led_ws2812/boards/nrf51dk_nrf51822.overlay`.
-
-Some boards are already supported out of the box; see the :file:`boards`
-directory for this sample for details.
-
 The sample updates the LED strip periodically. The update frequency can be
 modified by changing the :kconfig:option:`CONFIG_SAMPLE_LED_UPDATE_DELAY`.
 
+If there is no chain-length property in the devicetree node,
+you need to set the number of LEDs in the :kconfig:option:`SAMPLE_LED_STRIP_LENGTH`
```

Comment:
needs CONFIG_ prefix

Reply:
@gmarull why is this not failing CI?

Reply:
Thank you for catching the error.
Fixed it.